app/views/admin/navigation_links/_edit_navigation_link_modal.html.erb

```diff
@@ -0,0 +1,51 @@
+<div onclick="toggleEditNavigationLinkModal('<%= link[:id] %>')" class="color-accent-warning-darker cursor-pointer">
```

Comment:
it looks like this is exactly like add except for the title adn the onclick, maybe they can be generalized?

Reply:
I'll do this in a subsequent PR. I've made a note of it.